omero-grid/omero/Dockerfile

```diff
@@ -1,34 +1,49 @@
-FROM mattdm/fedora-small
-MAINTAINER Simon Li "spli@dundee.ac.uk"
+FROM centos:centos7
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-*-x86_64
-RUN yum -y install unzip java-1.8.0-openjdk ice ice-python ice-java postgresql
+ENV OMERO_VERSION 5.2.0
 
-# Optional dependencies
-RUN yum -y install python-tables python-pillow python-matplotlib numpy scipy
+RUN yum -y install epel-release && \
+    curl -o /etc/yum.repos.d/zeroc-ice-el7.repo \
+        http://download.zeroc.com/Ice/3.5/el7/zeroc-ice-el7.repo && \
+    yum -y install \
+        unzip wget patch \
+        java-1.8.0-openjdk \
+        ice ice-python ice-servers \
+        python-pip \
+        numpy scipy python-matplotlib python-pillow python-tables \
+        postgresql && \
+    yum clean all
 
-ENV OMERODATA /OMERO
+RUN pip install 'Django<1.9' omego
 
-RUN useradd omero
+RUN useradd omero && \
+    mkdir /OMERO && \
+    chown omero /OMERO
 
-ADD http://downloads.openmicroscopy.org/omero/5.0.1/artifacts/OMERO.server-5.0.1-ice35-b21.zip /home/omero/
-#ADD OMERO.server-5.0.1-ice35-b21.zip /home/omero/
-RUN chmod a+r /home/omero/OMERO.server-5.0.1-ice35-b21.zip
-RUN su -l - omero -c "cd /home/omero && \
-    unzip OMERO.server-5.0.1-ice35-b21.zip && \
-    ln -s OMERO.server-5.0.1-ice35-b21 OMERO.server"
+USER omero
+WORKDIR /home/omero
+RUN omego download server --release $OMERO_VERSION && \
```

Comment:
This will likely be the first thing that I'd need to change. Can we make all the arguments passable? Or unpack to a well-known location so an unzipped server can be mounted?